sdk/communication/communication-sms/samples/v1/typescript/src/usingAadAuth.ts

```diff
@@ -2,40 +2,27 @@
 // Licensed under the MIT Licence.
 
 /**
- *   @summary (ONLY AVAILABLE IN NODE.JS RUNTIME) Use AAD token credentials when sending a SMS message.
+ *   @summary Use AAD token credentials when sending a SMS message.
  */
 
-/*
-  ONLY AVAILABLE IN NODE.JS RUNTIME
-  If you are using the browser, you can use the InteractiveBrowserCredential provided via @azure/identity or any other feasible implementation of TokenCredential.
-
-  Setup :
-    Please ensure that your Communication Services resource is in US East, US East 2, or West Europe
-    region. AAD Role Based Access Control is not supported in other regions yet.
-
-    Register a new application in AAD
-      - See https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app
-        to register a new application in the Azure Active Directory.
-      - Note down the CLIENT_ID and TENANT_ID from the above step.
-      - In the "Certificates & Secrets" tab, create a secret and note that down.
-      - In the Azure portal, go to your Communication Services resource and click on the Access control (IAM)
-        tab. Here, assign the "Owner" role to the registered application.
-      
-    - Environment setup for the sample
-      - From the overview page of your AAD Application, note down the `CLIENT ID` and `TENANT ID`. In the "Certificates & Secrets" tab, create a secret and note that down.
-      - Make sure you have AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET as environment variables to successfully execute the sample(Can leverage process.env).
-*/
-
-import { SmsClient } from "@azure/communication-sms";
-import { DefaultAzureCredential } from "@azure/identity";
+import { parseConnectionString } from "@azure/communication-common";
+import { SmsClient, SmsSendRequest } from "@azure/communication-sms";
+import { isNode } from "@azure/core-http";
+import { ClientSecretCredential, DefaultAzureCredential, TokenCredential } from "@azure/identity";
 
 // Load the .env file if it exists
 import * as dotenv from "dotenv";
 dotenv.config();
 
 export async function main() {
+  console.log("== Send SMS Message With AAD Authentication ==");
+
+  // You will need to set this environment variable or edit the following values
   const endpoint =
-    process.env["COMMUNICATION_ENDPOINT"] || "https://<resource-name>.communication.azure.com";
+    process.env.COMMUNICATION_ENDPOINT ||
```

Comment:
Do we want to just remove this `process.env.COMMUNICATION_ENDPOINT`? We don't even have a reference to it in the `sample.env`.

Reply:
I'll add `COMMUNICATION_ENDPOINT` to sample.env because if you're using aad to authenticate you most likely won't have a connection string to parse. Also, if a developer has a connection string they might as well use it to authenticate instead of aad. We only have connection string in this sample because we don't have `COMMUNICATION_ENDPOINT`.